python/pyspark/ml/wrapper.py

```diff
@@ -263,7 +282,14 @@ def _fit_java(self, dataset):
 
     def _fit(self, dataset):
         java_model = self._fit_java(dataset)
-        return self._create_model(java_model)
+        model = self._create_model(java_model)
+
+        # SPARK-10931: This is a temporary fix to allow models to own params
+        # from estimators. Eventually, these params should be in models through
+        # using common base classes between estimators and models.
+        model._create_params_from_java()
```

Comment:
This might be better to move to `JavaModel.__init()` for the case of creating a model without fitting - e.g. `CountVectorizerModel` from vocabulary.